### PR TITLE
Db diagram

### PR DESCRIPTION
**Pull request by MostafaShahat2021**
## In this PR:
-⭐️Created DataBase from the diagram:
image
-⭐️Create table for many-to-many relationships.

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+CREATE TABLE invoice_items (
+  id INT GENERATED ALWAYS AS IDENTITY,
+  unit_price NUMERIC(5,2),
+  quantity INT,
+  total_price NUMERIC(5, 2),
+  invoice_id INT,
+  treatment_id INT,
+  PRIMARY KEY(id),
+  CONSTRAINT fk_invoice FOREIGN KEY(invoice_id) REFERENCES invoices(id),
+  CONSTRAINT fk_treatment FOREIGN KEY(treatment_id) REFERENCES treatments(id)
```
Comment (1GORDON) [inline schema_based_on_diagram.sql:53]: Great job team 🤝

- As stated in the project requirements, kindly ensure all FOREIGN KEYS have indexes created.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @MostafaShahat2021, and @Titus-Kiplagat ✋

Good job so far!

## Highlights!
- All tables have a PRIMARY KEY set✔️ 
- All "join tables" for the many-to-many relationships are created ✔️
- All "join table" for the many-to-many relationships is created ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by MostafaShahat2021** [comment]
Hey @1GORDON Thanks a lot for your efforts in reviewing our code

**Review by aamirkhan2478 — APPROVED** [review]
Hi @MostafaShahat2021 & @Titus-Kiplagat, 

GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ You used Gitflow correctly :+1: 
✅ No linter error :+1: 
✅ Good commit messages :+1: 
✅ Good descriptive PR :+1: 
✅ Well done :clap: you did all changes given by the previous reviewer :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by MostafaShahat2021** [comment]
Hey @aamirkhan2478 Thanks a lot 💯 for your efforts in reviewing our code, we really appreciate it 😄

**Comment by aamirkhan2478** [comment]
> Hey @aamirkhan2478 Thanks a lot 💯 for your efforts in reviewing our code, we really appreciate it 😄 

@MostafaShahat2021 my pleasure bro

**Pull request merged**